tests/system_tests_ssl.py

```diff
@@ -452,6 +458,76 @@ def get_expected_tls_result(self, expected_results):
                 self.OPENSSL_ALLOW_TLSV1_2 and tlsv1_2,
                 self.OPENSSL_ALLOW_TLSV1_3 and tlsv1_3]
 
+    def inject_file(self, source_file, dest_file):
+        source_path = self.ssl_file(source_file)
+        url         = Url("amqp://0.0.0.0:%d" % self.PORT_CLEAR)
+        try:
+            connection = BlockingConnection(url, sasl_enabled=False, timeout=self.TIMEOUT)
+            sender     = connection.create_sender('_$qd.store_file')
+            fd         = open(source_path, "r")
+            body       = fd.read()
+            fd.close()
+            sender.send(Message(subject=dest_file, body=body))
```

Comment:
Instead of calling this inject_file three times, why not send a message with a body as a dict.

Reply:
That would definitely be the right way to go for a production tool.  This was an easy way to upload those files for the test.  I didn't feel the need to overly optimize the test code.